skimage/evaluate/simple_metrics.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+from ..util.dtype import dtype_range
+from .._shared.utils import warn
+
+__all__ = ['mean_squared_error',
+           'normalized_root_mse',
+           'peak_signal_noise_ratio',
+           ]
+
+
+def _assert_compatible(im1, im2):
```

Comment:
Is this the ideal place for this function?  Would it make more sense for this to live in `_shared`?

Some duplicated effort where we ask for masks in other functions could be simplified with this.  I wouldn't mind removing the leading underscore in that case.

Reply:
Agreed, moved to `_shared` and renamed `assert_equal_shape`